modules/git/command.go

```
@@ -95,6 +95,18 @@ func (c *Command) AddArguments(args ...string) *Command {
 	return c
 }
 
+// AddDynamicArguments adds new dynamic argument(s) to the command.
+// If the argument is invalid (it shouldn't happen in real life), it panics to caller
+func (c *Command) AddDynamicArguments(args ...string) *Command {
+	for _, arg := range args {
+		if arg != "" && arg[0] == '-' {
+			panic("invalid argument: " + arg)
```

Why panic instead of returning an error?
I don't think we can ensure at compile time that no invalid command is given…

---

There are some sensitive information which can not be exposed at the moment.

The code works as it's purpose. Private me on discord.